src/app/components/navbar/navbar.component.ts

```diff
@@ -1,15 +1,23 @@
 import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute, NavigationEnd } from '@angular/router';
 
 @Component({
   selector: 'app-navbar',
   templateUrl: './navbar.component.html',
   styleUrls: ['./navbar.component.scss']
 })
 export class NavbarComponent implements OnInit {
+  currentRoute = '';
```

Comment:
Please specify the scope of the variable (public, private etc.)

Reply:
Done